pkg/gui/controllers/helpers/merge_and_rebase_helper.go

```diff
@@ -224,8 +225,10 @@ func (self *MergeAndRebaseHelper) RebaseOntoRef(ref string) error {
 			Key:   's',
 			OnPress: func() error {
 				self.c.LogAction(self.c.Tr.Actions.RebaseBranch)
-				err := self.c.Git().Rebase.RebaseBranch(ref)
-				return self.CheckMergeOrRebase(err)
+				return self.c.WithLoaderPanel(self.c.Tr.RebaseWait, func(task gocui.Task) error {
```

Comment:
Let's use `WithWaitingStatus` instead as that's the pattern we use for things that don't involve talking to a remote

Reply:
I think I've read in one of your comments in other issue to use something different that `WithWaitingStatus` , but I was lazy to search it again.  Now I have changed it, it is a spinner at the left bottom corner of the UI.